var/spack/repos/builtin/packages/tamaas/package.py

```diff
@@ -17,6 +17,7 @@ class Tamaas(SConsPackage):
     maintainers = ["prs513rosewood"]
 
     version("master", branch="master")
+    version("2.5.0.post1", sha256="9c550b3a905bfc973f79fc52c56a55cb2234087c8014e0177610ec8b76b46a71")
```

Comment:
Are you sure this is the correct sha256?  `spack checksum tames 2.5.0.post1` is giving me:

```suggestion
    version("2.5.0.post1", sha256="28e52dc5b8a5f77588c73a6ef396c44c6a8e9d77e3e4929a4ab07232dc9bc565")
```

Reply:
You are correct. Not sure how I got the hash wrong, I tested the installation before submitting.